exchange/exchange-ps/exchange/Set-ThrottlingPolicyAssociation.md

```diff
@@ -32,11 +34,11 @@ Set-ThrottlingPolicyAssociation [-Identity] <ThrottlingPolicyAssociationIdParame
 ## DESCRIPTION
 The Set-ThrottlingPolicyAssociation cmdlet defines quota limits for specific objects. For example, if you notice that a user or other object is using excessive bandwidth, you can associate that object with a throttling policy that's more restrictive.
 
-In data center deployments, the object referred to by the Identity and ThrottlingPolicy parameters must be in the same tenant.
+In data center deployments, the Identity parameter-referred object and the ThrottlingPolicy parameter-referred object must be in the same tenant.
```

Comment:
Please retain the previous sentence as is..your proposed change, changes the meaning of the sentence itself and hence seem to be inaccurate